src/compiler/types.ts

```diff
@@ -2645,9 +2645,8 @@ namespace ts {
          * Does not include properties of primitive types.
          */
         /* @internal */ getAllPossiblePropertiesOfType(type: Type): Symbol[];
-
+        /* @internal */ resolveName(name: string, location: Node, meaning: SymbolFlags): Symbol;

```

Comment:
The compiler doesn't have `strictNullChecks` on. Added `| undefined` to be explicit.